Source/ImPlotWidgets/Public/ImPlotWidget.h

```diff
@@ -10,24 +10,27 @@ struct FDataPoint
 	float Y = 0.0f;
 };
 
+struct FImPlotData
+{
+	char* Name;
```

Comment:
I would change this from a char* to an FString now to fit in more with the Unreal way of doing things